src/prefect/tasks/prefect/flow_run_cancel.py

```diff
@@ -0,0 +1,38 @@
+from prefect import Task
+from prefect.client import Client
+from typing import Any
+import prefect
+
+
+class CancelFlowRunTask(Task):
+    """
+    Task to cancel a flow.
+    Args:
+        - flow_run_id_to_cancel (str, optional): The ID of the flow to cancel
+        - **kwargs (dict, optional): additional keyword arguments to pass to the Task constructor
+    """
+
+    def __init__(
+        self,
+        flow_run_id_to_cancel: str = None,
+        **kwargs: Any,
+    ):
+        self.flow_run_id_to_cancel = flow_run_id_to_cancel
+        super().__init__(**kwargs)
+
+    def run(self, flow_run_id_to_cancel: str = None):
+        """
+        Args:
+            - flow_run_id_to_cancel (str, optional): The ID of the flow to cancel
+        Returns:
+            - bool: Boolean representing whether the flow was canceled successfully or not
```

Comment:
We had a long debate about this when introducing the state if it should be canceled or cancelled 😉  (cc @cicdw)